src/services/database/index.js

```diff
@@ -0,0 +1,44 @@
+import schemas from './schemata';
+
+const Realm = require('realm');
```

Comment:
Can we use import here instead of require? It gives more consistency with other project parts.

Reply:
done